web/packages/teleport/src/services/localStorage/types.ts

```diff
@@ -34,12 +34,21 @@ export const KeysEnum = {
 export type SurveyRequest = {
   companyName: string;
   employeeCount: string;
-  resources: Array<string>;
+  resourcesList: Array<string>;
```

Comment:
think it's called this elsewhere? (it's also more clear)

```suggestion
  preferredResources: Array<string>;
```

Reply:
This is inline the the Sales Center request to `SetSurveyResultsRequest` from the tenants proto. We refer to it as preferred resources in the cluster state; but on the account.metadata.survey its 'resources' and the generated pb file makes that repeated list "resourcesList"

I can imagine renaming these to be in line with the cluster state key; but I would address that separately since we'll need changes in Sales Center.